wiki/Client/Beatmap_editor/Compose/en.md

```diff
@@ -1,351 +1,212 @@
+![](img/playfield.jpg "Visual representation of the current timestamp")
 
-### Centre-Right (Hitsounds & Assist Tools)
+Difficulties are mapped here and during gameplay, they will appear exactly like displayed in the editor. For fine-grained mapping, the `.osu` file can be directly edited using a text editor like Notepad.
 
-**List of available subtools complement to the original DIY toolkit.**
+### Subtools
 
-![Hitsounds & Assist Tools](img/BE_HSAT.jpg "Hitsounds & Assist Tools")
+![](img/subtools.jpg "List of available subtools")
 
-| Button (Keyboard Shortcut) | Usage | Description |
+| Button (Keyboard shortcut) | Usage | Description |
 | :-- | :-- | :-- |
-| New Combo (`Q`) | **Right-click:** Change current note to next combo colour. | Note combo. This is equivalent to a verse / measure in song / instrument script. An acceptable range is about 5-20. |
+| `New Combo` (`Q`) | `Right click`: Change current object to next combo colour. | Hit object combo. This is equivalent to a verse in song script. An acceptable range is about 5–20. |
 
 #### Hitsounds
 
-| Button (Keyboard Shortcut) | Usage | Description |
+| Button (Keyboard shortcut) | Usage | Description |
 | :-- | :-- | :-- |
-| Whistle (`W`) | ("Select" on a Hit note) **Left-Click:** Use whistle sound on this note | Add a whistle sound addition to the selection. |
-| Finish (`E`) | ("Select" on a Hit note) **Left-Click:** Use finish sound on this note | Add a finish (cymbal) sound addition to the selection. |
-| Clap (`R`) | ("Select" on a Hit note) **Left-Click:** Use clap sound on this note | Add a clap sound addition to the selection. |
+| `Whistle` (`W`) | **(Select a hit object)** `Left click`: Use whistle sound on this object. | Add a whistle sound addition to the selection. |
+| `Finish` (`E`) | **(Select a hit object)** `Left click`: Use finish sound on this object. | Add a finish (cymbal) sound addition to the selection. |
+| `Clap` (`R`) | **(Select a hit object)** `Left click`: Use clap sound on this object. | Add a clap sound addition to the selection. |
 
-#### Assist Tools
+#### Assist tools
 
-| Button (Keyboard Shortcut) | Usage | Description |
+| Button (Keyboard shortcut) | Usage | Description |
 | :-- | :-- | :-- |
-| Grid Snap (`T`) | **`Shift` (Hold):** Temporary toggle. **`Ctrl` (Hold):** Temporary disable angle snapping. | Hit notes will be snapped to the grid by default. |
-| Distance Snap (`Y`) | **`Alt` (Hold):** Temporary toggle. Switch Beat Snap Divisor to Distance Snap **`Alt` + Mousewheel:** Adjust Distance Snap multiplier. | Distance between consecutive hit notes will be snapped based on their rhythmical difference. Best use while the timeline is paused. |
-| Lock Notes (`L`) | ("Select" on a Hit note) **Left-Click:** Lock this note | Lock selected note to current position and timestamp. |
+| `Grid Snap` (`T`) | **Hold** `Shift`: Temporary toggle. **Hold** `Ctrl`: Temporarily disable angle snapping. | Hit objects will be snapped to the grid by default. |
```

Comment:
looks like the `Ctrl` piece refers to functionality where you could draw sliders tilted by multiples of 45° by holding `Ctrl`. it's either removed from the editor, or I forgot how to actually trigger it -- if it's the former, please remove this part